radionice/srednji/lekcije/030_kauzativ/030_kauzativ.tex

```diff
@@ -8,20 +8,20 @@
 	
 	\begin{table}[h]
 		\centering
-		\begin{tabular}{l l l l}\toprule[2pt]
+		\begin{tabular}{l l}\toprule[2pt]
```

Comment:
Nedostaju naslovi stupaca. Ako imaš neki bolji način da prostorno složiš tablicu, to bi bilo super. Visoka je, a uska pa trati papir.